### ISSUE TITLE
Soft reset leads to C8 error

### ISSUE THREAD

**Issue by RamMan21** [ClusterM/hakchi2#898]
When I soft reset from snes rom it works perfectly. when i attempt from nes rom, it leads to C8 error.

A hard reset from Nes rom leads to C6 error but wont lock up the console.

Am I the only one with this issue?

**Comment by SeeDborg** [comment]
I have the same issue.

**Comment by ryandoll** [comment]
Same thing for me too. Gameboy and NES games will prompt errors, but using either the controller shortcut or reset button on any SNES game works fine.

**Comment by Drakonas** [comment]
what console? Classic NES or Classic SNES? Also, are you using a dual boot method, or are you using retroarch for the alternate console? What games are you having issues with?

**Comment by ryandoll** [comment]
Classic SNES. Retro arch I assume for trying to play the NES games, and whatever the gameboy one is (I have both of those mods installed) no dual boot.

**Comment by Drakonas** [comment]
Did you install the retroarch module as well as the cores?

**Comment by ryandoll** [comment]
Here is what is installed:
![capture](https://user-images.githubusercontent.com/1770175/31924859-10b52042-b84a-11e7-9cb1-995421eed11d.PNG)
Any NES Game (Batman was one I tried) or any Original GameBoy game (Super Mario Land 2 - 6 Golden Coins) will prompt the issue when trying to do a reset via the controller command (Currently set to UP and select) or using the reset button on the SNES Classic.
![img_2494](https://user-images.githubusercontent.com/1770175/31924960-d2caea68-b84a-11e7-8827-055978b9810b.JPG)

**Comment by Drakonas** [comment]
@ryandoll You have no nes core installed. You need the nestopia or fceumm hmod to use NES roms. The SNES has no software included for running NES games.

**Comment by Drakonas** [comment]
@RamMan21 is your problem resolved? Please close this ticket if so.

**Comment by RamMan21** [comment]
@ryandoll the issue was with clovercon not installed. try to reinstall it.

**Comment by ryandoll** [comment]
Don’t understand. The games play just fine, just can’t soft reset then. I will try to reinstall tonight.

**Comment by Drakonas** [comment]
@ryandoll How are you soft resetting? Are you holding down and select on your controller?

**Comment by ryandoll** [comment]
That or the reset button on the SNES Classic.  To be fair, maybe something didn't update properly when I set up my hakchi2 directory.  I was using it for my NES classic...saw there was an update to support the SNES classic, dropped in the newer updated files into the same hakchi2 directory on my computer, plugged in my SNES Classic and went through the normal prompts.  Maybe there is an old mod that is out of sync by doing it this way?  I will try to reinstall tonight when I get home.  Thanks for all the help!

**Comment by Drakonas** [comment]
That would make hakchi use configs for NES Classic for the SNES Classic. DON'T DELETE YOUR DUMP FOLDER! It has your OG kernel in it!

**Comment by ryandoll** [comment]
Yep, I backed those up already.  Cool, let me try again.  Thanks for the patience.
